### PR TITLE
Email functionality / Username change

### PR DESCRIPTION
**Pull request by leong96**
This commit implements the email functionality requested in Slack, as well as implements the username change feature. This should handle #90 and #134.

functions.py: Moved the change username function to functions.py, saves us a couple lines of code in views.

admin.php: hooked up the change username form to the backend.

allAdmins: Hooked up the create admin form to the backend.

allusers: Hooked up the invite provider form to the backend.

provider: hooked up the change email form to the backend.

views: Added all calls to send_email to send emails in the situations specified in Slack.

### CODE REVIEW

File: grasa_event_locator/views.py [hunk 1]
```diff
@@ -343,6 +347,7 @@ def denyEvent(request, eventID):
         if request.user.is_authenticated and request.user.userinfo.isAdmin and request.user.userinfo.isActive:
                 p = Program.objects.get(pk=eventID)
                 p.delete()
+                print(send_email([str(User.objects.get(pk=p.user_id.user_id))], "GRASA - Event Denied", "Your edited event has been denied. Contact GRASA for details."))
```
Comment (R-Eli-Levasseur) [inline grasa_event_locator/views.py]: This says your edited event,  should just be your event

File: grasa_event_locator/templates/provider.php [hunk 2]
```diff
@@ -38,7 +38,7 @@
                         <form action="provider.php" method="post">
                             {% csrf_token %}
                              <div class="input-group mb-3 changeNameInput">
-                                <input type="text" class="form-control" placeholder="{{ user.userinfo.org_name }}" aria-label="{{ user.userinfo.org_name }}" value="{{ user.userinfo.org_name }}" name="changename">
+                                <input type="text" name="changename" class="form-control" placeholder="{{ user.userinfo.org_name }}" aria-label="{{ user.userinfo.org_name }}" value="{{ user.userinfo.org_name }}" name="changename">
```
Comment (R-Eli-Levasseur) [inline grasa_event_locator/templates/provider.php]: name="changename" is in here twice

File: grasa_event_locator/views.py (hunk 1 above)
Comment (leong96) [reply grasa_event_locator/views.py]: Altered in a subsequent commit.

File: grasa_event_locator/templates/provider.php (hunk 2 above)
Comment (leong96) [reply grasa_event_locator/templates/provider.php]: Also corrected in a subsequent commit.

**Review by R-Eli-Levasseur — APPROVED** [review]

**Pull request merged**